javascript/javascript2/week1/Js2Week1.js

```diff
@@ -0,0 +1,102 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const lengthArr = [];
+let min;
+let shortestWord;
+
+function findShortestWord() {
```

Comment:
I would name this getShortestWord so the name illustrates exactly what it does. This is a common naming convention when a function returns something

Reply:
Will pay more attention to the naming convention the next time